recipes/obspy/run_test.py

```diff
@@ -0,0 +1,9 @@
+import matplotlib
+matplotlib.use("AGG")
+from obspy.core import run_tests
+errors = run_tests(report=True, hostname="conda-forge", verbosity=2)
```

Comment:
> If you can live with the way the tests are run now it is fine. 

Yes, works like a charm.

> Somehow I had the impression that you would need some extra workarounds.

No, it's fine. I can set an env variable in `run_tests.py` myself right before the tests are run and use it as a condition to skip the stray known fail in our test suite.

> PS: Travis-CI failure was due to a bad connection. I just re-started and it for you.

Yep, saw that. Thanks for restarting